src/Helpers/SavesAndFollows/Follow.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ActionType,
+  ContextModule,
+  FollowedArtist,
+  OwnerType,
+  UnfollowedArtist,
+} from "../../Schema"
+
+export interface FollowedArtistArgs {
+  contextModule: ContextModule
+  contextOwnerId?: string
```

Comment:
I see they're also not used here. Interesting that exporting the interface doesn't need them.